.eslintrc

```diff
@@ -22,6 +22,7 @@
     },
     "globals": {
         "axs": true,
-        "$": true
+        "$": true,
+        "chrome": false
```

Comment:
This global is added for the Chrome extension scripts.

Reply:
I can actually move this global I to a .eslintrc file in the extension configuration folder.

Reply:
Done.